community-membership.md

```diff
@@ -39,6 +39,7 @@ remain active contributors to the community.
 ### Requirements
 
 - Enabled [two-factor authentication] on their GitHub account
+- Updated their affiliation and email in [CNCF gitdm]
```

Comment:
```suggestion
- Updated their affiliation (can be "independent") and email in [CNCF gitdm]
```

Also, what if they choose not to divulge an email address through the noreply feature? I know people who choose not to give that out or even post it publicly in a modified form due to stalking concerns.